src/test/resources/allure.properties

```diff
@@ -0,0 +1 @@
+allure.results.directory=target/allure-results
```

Comment:
in pom.xml you specified build folder `${project.build.directory}/allure-results`. To be consistent can you also specify it here instead of target one? Or maybe you even do not need `allure.properties` at all if you configured allure via pom.xml (need to check it)

Reply:
I've looked into it and modified the pom.xml file.